packages/liquidator/test/Liquidator.js

```diff
@@ -33,16 +33,39 @@ const SyntheticToken = artifacts.require("SyntheticToken");
 const Timer = artifacts.require("Timer");
 const Store = artifacts.require("Store");
 
+// Run the tests against 3 different kinds of token/synth decimal combinations:
```

Comment:
As with the emp client we accommodate all 3 types of combinations within the tests.